src/error_handling.rs

```diff
@@ -25,7 +25,7 @@ macro_rules! error_msg {
         let msg = format!($($arg)+);
         log::error!("{}", msg);
         EVENT_AGGREGATOR.send(UiCommand::Parallel(ParallelCommand::ShowError {
-            lines: msg.split('\n').map(|s| s.to_string()).collect_vec(),
+            lines: itertools::Itertools::collect_vec(msg.split('\n').map(|s| s.to_string())),
```

Comment:
Was this change intentional? It only makes things longer. If the intention was to avoid the `Itertools` dep, this could be done using `.collect::<Vec<_>>()`.

Reply:
It was intentional to avoid requiring a using statement on the user side of the macro.